<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow
{
background-color: qlineargradient(spread:pad, x1:0.5285, y1:1, x2:0.505, y2:0, stop:0 rgba(0, 57, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>801</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ravie</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#11c8ff;&quot;&gt;TEAM ICE&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="imageProcessingButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>150</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Image 
Processing</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="impedanceButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>150</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Impedance
Graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minimizeButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>150</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Close
Program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shutdownButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>320</y>
      <width>150</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Shutdown
RPi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openPortButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>150</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Open/Close
Serial Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colonyCountGraph">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>150</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Colony Count
Graph</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
